config/initializers/datadog.rb

```diff
@@ -21,7 +19,8 @@
   c.tracing.instrument :rails
   c.tracing.instrument :elasticsearch
   c.tracing.instrument :shoryuken
+  c.tracing.instrument :graphql, enabled: false, with_deprecated_tracers: true
```

Comment:
This is trying to disable the graphql tracer because it's using the old version and it throws a compatability warning.